libs/application/templates/parental-leave/src/forms/OtherParentApproval.ts

```diff
@@ -7,29 +7,70 @@ import {
   Form,
   FormModes,
   coreMessages,
+  buildTextField,
+  Application,
 } from '@island.is/application/core'
 
 import Logo from '../assets/Logo'
+import { YES } from '../constants'
 import { otherParentApprovalFormMessages } from '../lib/messages'
+import { getApplicationAnswers } from '../parentalLeaveUtils'
 
 export const OtherParentApproval: Form = buildForm({
   id: 'OtherParentApprovalForParentalLeave',
-  title: otherParentApprovalFormMessages.formTitle,
+  // title: otherParentApprovalFormMessages.formTitle,
+  title: 'Tilfærsla réttinda',
   logo: Logo,
   mode: FormModes.REVIEW,
   children: [
     buildSection({
       id: 'review',
-      title: otherParentApprovalFormMessages.reviewSection,
+      // title: otherParentApprovalFormMessages.reviewSection,
+      title: 'Samþykki',
       children: [
         buildMultiField({
           id: 'multi',
-          title: otherParentApprovalFormMessages.multiTitle,
+          // title: title: otherParentApprovalFormMessages.multiTitle,
+          title: 'Samþykki á tilfærslu réttinda',
           children: [
             buildDescriptionField({
               id: 'intro',
               title: '',
-              description: otherParentApprovalFormMessages.introDescription,
+              // description: otherParentApprovalFormMessages.introDescription,
+              description:
+                'Í umsókn um fæðingarorlof hefur hitt foreldrið óskað eftir tilfærslu á réttindum frá þér til sín. Óskað er eftir tilfærslu á eftirfærandi réttindum:',
+            }),
+            buildTextField({
+              id: 'requestingRights',
+              title: 'Fjöldi daga sem óskað er eftir',
+              description:
+                'Að nýta daga af þínum rétti til töku fæðingarorlofs, við það minnkar hámarksréttur þinn',
+              condition: (answers) =>
+                getApplicationAnswers(answers).isRequestingRights === YES,
+              defaultValue: 45,
+              disabled: true,
+            }),
+            buildTextField({
+              id: 'requestingPersonalDiscount',
+              title: 'Hlutfall sem óskað er eftir',
+              description:
```

Comment:
Intentionally hard coded to not clash with locale strings in progress and can be integrated